packages/terra-form-field/src/Field.scss

```diff
@@ -24,7 +24,7 @@
   }
 
   .label {
-    color: var(--terra-form-field-label-color, rgba(0, 0, 0, 0.8));
+    color: var(--terra-form-field-label-color, #1c1f21);
```

Comment:
What are these stylistic changes for?

Reply:
Talked to @neilpfeiffer about accessibility issues with help text and error text colors and based on suggestions from UX, had to change the color of label too.